### PR TITLE
added test cases: Spanish, Italian, French

### PR DESCRIPTION
**Pull request by AshleyBellomy1**
## Pull Request Checklist

Thank you for taking the time to improve Arrow! Before submitting your pull request, please check all *appropriate* boxes:

- [x ] 🧪  Added **tests** for changed code.
- [ x] 🛠️  All tests **pass** when run locally (run `tox` or `make test` to find out!).
- [x ] 🧹  All linting checks **pass** when run locally (run `tox -e lint` or `make lint` to find out!).
- [x ] 📚  Updated **documentation** for changed code.
- [x ] ⏩  Code is **up-to-date** with the `master` branch.

If you have *any* questions about your code changes or any of the points above, please submit your questions along with the pull request and we will try our best to help!

### CODE REVIEW

File: tests/test_locales.py [hunk 1]
```diff
@@ -177,14 +177,118 @@ def test_meridian_invalid_token(self):
...
+        assert self.locale.ordinal_number(113) == "113º"
+        assert self.locale.ordinal_number(114) == "114º"
+        assert self.locale.ordinal_number(121) == "121º"
+        assert self.locale.ordinal_number(122) == "122º"
+        assert self.locale.ordinal_number(123) == "123º"
+        assert self.locale.ordinal_number(124) == "124º"
 
+    def test_format_timeframe(self):
+        assert self.locale._format_timeframe("now", 0) == "adesso"
+        assert self.locale._format_timeframe("seconds", 1) == "un secondo"
```
Comment (anishnya) [inline tests/test_locales.py:215]: This should be ```self.locale._format_timeframe("second", 1)```. We use ```self.locale._format_timeframe("second", 1) ``` for a string that represents one second.

File: tests/test_locales.py [hunk 2]
```diff
@@ -211,34 +315,110 @@ def test_format_timeframe(self):
...
-        assert self.locale._format_timeframe("week", -1) == "una semana"
-        assert self.locale._format_timeframe("weeks", -2) == "2 semanas"
-        assert self.locale._format_timeframe("weeks", -3) == "3 semanas"
-        assert self.locale._format_timeframe("month", -1) == "un mes"
-        assert self.locale._format_timeframe("months", -3) == "3 meses"
-        assert self.locale._format_timeframe("months", -13) == "13 meses"
-        assert self.locale._format_timeframe("year", -1) == "un año"
-        assert self.locale._format_timeframe("years", -4) == "4 años"
-        assert self.locale._format_timeframe("years", -14) == "14 años"
+        assert self.locale._format_timeframe("seconds", -1) == "hace 1 segundos"
```
Comment (anishnya) [inline tests/test_locales.py:318]: ```self.locale._format_timeframe``` doesn't handle the relative logic, hence why these tests are failing. If you wanted to check whether strings are properly being formatted in their future or past form, that would be done with ```self.locale._format_relative```

File: tests/test_locales.py [hunk 3]
```diff
@@ -211,34 +315,110 @@ def test_format_timeframe(self):
...
+        assert self.locale.ordinal_number(113) == "113e"
+        assert self.locale.ordinal_number(114) == "114e"
+        assert self.locale.ordinal_number(121) == "121e"
+        assert self.locale.ordinal_number(122) == "122e"
+        assert self.locale.ordinal_number(123) == "123e"
+        assert self.locale.ordinal_number(124) == "124e"
+
+    def test_format_timeframe(self):
+        assert self.locale._format_timeframe("now", 0) == "maintenant"
+        assert self.locale._format_timeframe("seconds", 1) == "une seconde"
```
Comment (anishnya) [inline tests/test_locales.py:378]: See comment above about using ```self.locale._format_timeframe("second", 1)```

File: tests/test_locales.py [hunk 4]
```diff
@@ -211,34 +315,110 @@ def test_format_timeframe(self):
...
+        assert self.locale._format_timeframe("years", -4) == "hace 4 años"
+        assert self.locale._format_timeframe("years", -14) == "hace 14 años"
 
 
 @pytest.mark.usefixtures("lang_locale")
 class TestFrenchLocale:
     def test_ordinal_number(self):
+
+
+        assert self.locale.ordinal_number(0) == "0ème"
```
Comment (anishnya) [inline tests/test_locales.py:346]: This is an edge case we're missing within ```_ordinal_number``` method for the FrenchBaseLocale in ```arrow\locales.py```. Feel free to add the appropriate edge case with this method to fix this.

**Review by anishnya — CHANGES_REQUESTED** [review]
Hey @AshleyBellomy1. I've left come comments on the PR that go over why the checks are failing. Feel free to ask any questions or clarification if they arise. Thanks for contributing to Arrow :)
